### ISSUE TITLE
Quick 7.11 Docs note to support Endpoint Security Telemetry feature Agent side requirements

### ISSUE THREAD

**Issue by EricDavisX** [elastic/observability-docs#318]
Hi.  @kevinlog mentions this to the Ingest Team, and we wanted to document it.  We will need to quickly confirm specifics to type up the docs.

https://github.com/elastic/kibana/pull/86277

This ticket merges a small change, about the Protections Team Telemetry feature, but I don't know off hand the larger meta ticket, sorry.  Kevin can you post and update?

Assigning Janeen and Dede to coordinate on which parts may go into one or both of Security side and Fleet / Agent side docs.

We know this:

if you upgrade an Agent from pre-7.11 the Telemetry feature will be on by default in the 7.11 Agent and it will attempt to write to the new index per the above issue, but it will fail (because the API key, as upgraded, will not have permissions).

Nothing else bad will happen, apart from errors in the logs, mind you.  but we expect users to notice and ask about it so we can be proactive.

The work arounds we have so far are this:
 - re-enroll the Agent to pick up the API key changes (which should be available automatically after upgrading, right Kevin?)
 - change the Kibana side policy for this Telemetry feature to turn it off so it won't keep logging error messages on Elastic Endpoint side as well as Elastic index side (likely 1 error per Elastic Endpoint per some time frame).  We don't know for sure what the back-off policy is on attempting this, so it could be a lot of noise in the Elastic and Endpoint side logs.  Kevin was going to log a ticket for improving that back-off strategy for this specific case (as needed).

**Comment by dedemorton** [comment]
@EricDavisX Sounds like this error appears in the Elastic Agent logs? If so, it probably makes sense for me to document this in the Elastic Agent troubleshooting docs.

Can you give me the message that users will see so I can craft a description that matches what users see? If you can respond soonish, I can probably still get this in before 7.11 drops.

**Comment by EricDavisX** [comment]
I don't have any data - I pinged @kevinlog in slack for help.

**Comment by kevinlog** [comment]
@dedemorton the relevant log will come in via the Endpoint logs which are accessible via Fleet.  Since the Endpoint speaks directly to ES, the Agent log wouldn't report anything in this case.

Here is an example from Endpoint logs:
`(action [indices:admin/auto_create] is unauthorized for API key id [KrvCi3YB96EBa5C9k2Ce] of user [fleet_enroll] on indices [.logs-endpoint.diagnostic.collection-default], this action is granted by the privileges [auto_configure,create_index,manage,all])`

You would be able to see this from the Fleet UI and selecting the Endpoint logs:
![image](https://user-images.githubusercontent.com/56395104/107064259-4fbf4080-67a9-11eb-9a93-fcd3364c2aef.png)

This indicates that the Endpoint does not have the correct permissions to send telemetry.

* If the user upgraded from `7.10` -> `7.11`, they will need to wait for `7.11.1` for a patch fix to ensure they get the correct permissions.  This is due to a permissions migration issue in Fleet.
* If it's a fresh `7.11` install, they will get the appropriate permissions
* See this issue for more info on the above bug: https://github.com/elastic/kibana/issues/87995

To turn off the feature entirely and get rid of the logs, they should edit the Endpoint Security integration.  Inside of the "Advanced" section, set `windows.advanced.diagnostic.enabled` to `false`

![image](https://user-images.githubusercontent.com/56395104/107064619-b80e2200-67a9-11eb-9ee9-a863aecb88c4.png)

Let me know if you have any questions on the above.

tldr: if a user was just looking at Agent logs, they likely wouldn't see anything related to this issue.  It would only show up in Endpoint logs.  If the user isn't using an Endpoint at all, this issue wouldn't surface.

**Comment by dedemorton** [comment]
@kevinlog Thanks so much for all the detail. I've added a troubleshooting topic in this PR: https://github.com/elastic/observability-docs/pull/374. Would appreciate your review. Thanks!
